diff --git a/src/Form/SharethisConfigurationForm.php b/src/Form/SharethisConfigurationForm.php
index 1745bad..f5cc47c 100644
--- a/src/Form/SharethisConfigurationForm.php
+++ b/src/Form/SharethisConfigurationForm.php
@@ -238,7 +238,7 @@ class SharethisConfigurationForm extends ConfigFormBase {
         '#description' => $this->t('Select which view modes the ShareThis widget should appear on for %label nodes.', array('%label' => $bundle_info['label'])),
         '#type' => 'checkboxes',
         '#options' => $modes,
-        '#default_value' => $config->get('sharethisnodes.' . $bundle),
+        '#default_value' => $config->get('sharethisnodes.' . $bundle) ?: [],
       );
     }
     // Allow the user to choose which content types will have ShareThis added
